src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -274,8 +274,7 @@ private void AddParameter(MamlParameter parameter, MamlCommand command)
 
             AddHeader(ModelTransformerBase.PARAMETERSET_NAME_HEADING_LEVEL, '-' + parameter.Name, extraNewLine: extraNewLine);
 
-            // for some reason, in the update mode parameters produces extra newline.
-            AddParagraphs(parameter.Description, this._mode == ParserMode.FormattingPreserve);
+            AddParagraphs(parameter.Description);
```

Comment:
This change is from a different PR, please use `git rebase` and use 1 branch per PR.